src/m_vector.h

```diff
@@ -37,13 +37,15 @@ typedef struct
 #if defined(HAVE_EXT_VECTOR_TYPE)
 typedef float quat __attribute__((ext_vector_type(4)));
 #else
-typedef struct
+#pragma pack(push, 1)
```

Comment:
Why are you only using the `#pragma pack` from MSVC and not the `PACKED_PREFIX` and `PACKED_SUFFIX` macros?

What is the benefit of setting the alignment here?

Reply:
Okay, but why do we need `ALIGNED`? I suppose it could theoretically improve performance if the compiler uses SIMD vector instructions. Did you measure the improvement?
For now, it seems like an unnecessary complication that might become obsolete with future CPUs or compilers.